src/constants/pools.ts

```diff
@@ -420,6 +421,7 @@ const POOLS_POLYGON: Pools = {
       '0xb20fc01d21a50d2c734c4a1262b4404d41fa7bf000000000000000000000075c',
       '0x8ac5fafe2e52e52f5352aec64b64ff8b305e1d4a0002000000000000000007ab',
       '0x4b3fd0347b7fb1b7bafa61da841e40d5bf6fe0f5000200000000000000000868',
+      '0x02d2e2d7a89d6c5cb3681cfcb6f7dac02a55eda400000000000000000000088f',
```

Comment:
I don't believe this pool is stakable, so please remove it from this list.
```suggestion
```

Reply:
Removed